### DISCUSSION TITLE
Test chatbot blank in Botfront interface

### DISCUSSION THREAD

**Discussion by castigliana** [botfront/botfront#1014] [General]
I've created a simple hello chatbot based on tutorial provided. The test chatbot within the botfront UI is blank. There is no welcome message coming back.

**Comment by Abu-Ishak** [comment]
Do you install Botfront on localhost or VM? If VM, then you have to set up a firewall for port 8888, 5050, 5005.

**Comment by castigliana** [comment]
That did it! Thank you Abu.
